Src/VimWpf/RelativeLineNumbers/LineNumberFormatTracker.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System;
+using System.Globalization;
+using System.Linq;
+using System.Windows.Media;
+using System.Windows.Media.TextFormatting;
+
+using Microsoft.VisualStudio.Text.Classification;
+using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.Text.Formatting;
+
+namespace Vim.UI.Wpf.RelativeLineNumbers
+{
+    internal class LineNumberFormatTracker : ILineFormatTracker
+    {
+        private readonly IWpfTextView _textView;
+        private readonly IFormattedLineSource _formattedLineSource;
+        private readonly IClassificationFormatMap _classificationFormatMap;
+        private readonly IClassificationTypeRegistryService _classificationTypeRegistry;
+
+        private TextFormattingRunProperties _formatting;
+
+        private TextFormatter _textFormatter;
+
+        private bool _formatChanged;
+
+        private bool _numbers;
+
+        private bool _relativeNumbers;

```

Comment:
Nit: don't have a huge preference if fields should be separated by blank lines or not (pattern in VsVim is generally no separation). But prefer within a class it's either but not both 😄

Reply:
Ok. I agree, every project has their own coding styles, so I'll put this into my .editorconfig for VsVim.
I'm just used from my last job to group fields into logical scopes.